apps/web/src/pages/tx/[hash].tsx

```diff
@@ -111,6 +112,14 @@ const Tx: NextPage = () => {
                     </Link>
                   ),
                 },
+                ...(txData.sourceRollup
+                  ? [
+                      {
+                        name: "Rollup Source",
+                        value: <RollupBadge rollup={txData.sourceRollup} />,
```

Comment:
where does this sourceRollup attribute come from?

Reply:
Is a new attribute on the table. This PR is built on top of the following change to the API: #277